### PR TITLE
Update redirects in Readme

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,13 +46,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ## Third-party services
 - [Atlassian Stash](https://marketplace.atlassian.com/plugins/be.foreach.stash.notifier) - Send notifications to a Slack channel when a repo push occurs
 - [Datadog](https://www.datadoghq.com/blog/collaborate-share-track-performance-slack-datadog/) - Send notifications to slack when alerts trigger, and share graphs with your colleagues in chat.
-- [Drupal](https://drupal.org/project/slack) - Send messages from a Drupal website to Slack
-- [Graylog2](http://www.graylog2.org/plugins/545cc0ace4b0d324cb87ad6d) - Send log alerts to Slack
+- [Drupal](https://www.drupal.org/project/slack) - Send messages from a Drupal website to Slack
+- [Graylog2](https://www.graylog.org/plugins/545cc0ace4b0d324cb87ad6d) - Send log alerts to Slack
```
Comment (matiassingers) [inline README.md]: this looks like it's broken now, can you update it to: https://github.com/Graylog2/graylog-plugin-slack

File: README.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [slack-assembla-plugin](https://github.com/Ramzi-Alqrainy/slack-assembla-plugin) - Watches Slack channels for messages about Assembla and acts accordingly
 - [slack-bill-sharing](https://github.com/gsellator/slack-bill-sharing) - Bill sharing bot for Slack
 - [slack-bot-api](https://github.com/mishk0/slack-bot-api) - Simple way to control your Slack Bot
-- [slack-brewbow](https://github.com/swirlycheetah/slack-brewbow) - Whose turn is it to make a brew? Let BrewBot decide
+- [slack-brewbow](https://github.com/swirlycheetah/slack-brewbot) - Whose turn is it to make a brew? Let BrewBot decide
```
Comment (matiassingers) [inline README.md]: Can we update `slack-brewbow` to `slack-brewbot` as well?

File: README.md [hunk 3]
```diff
@@ -251,15 +251,15 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [slack-webex-plugin](https://github.com/gsingers/slack-webex-plugin) - Launch a Webex Personal Room or other type of meeting from Slack
 - [slack-yandex-api](https://github.com/wearegoat/slack-yandex-api) - Simple server app for Yandex API and Slack integration
 - [slack.com.js](https://github.com/caseman72/slack.com.js) - Google Chrome extension to make emoticons look better
-- [slack.com.js](https://github.com/caiosba/slack.com.js) (II) - Google Chrome extension to turn issues numbers into links
+- [slack.com.js](https://github.com/caiosba/slack-issue-tracker-parser) (II) - Google Chrome extension to turn issues numbers into links
```
Comment (matiassingers) [inline README.md]: Can we update this to `slack-issue-tracker-parser`?

Comment (matiassingers) [inline README.md]: And update the description: "A Google Chrome extension to turn issues numbers into links to the issue on an issue tracker"

File: README.md [hunk 4]
```diff
@@ -357,8 +357,8 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [django-slack](https://github.com/lamby/django-slack) - Slack integration for Django, using the templating engine to generate messages
 - [django-utils](https://github.com/themotleyfool/django-utils) - Django utilities
 - [django-slack-oauth](https://github.com/izdi/django-slack-oauth) - Slack OAuth module for Django
-- [edie-slack](https://github.com/christinac/edie-slack) - Eliza's younger, hipper, psychobabble-friendly successor as a Slackbot
-- [err](https://github.com/gbin/err) - A plugin-based chatbot designed to be easily deployable, extensible and maintainable
+- [edie-slack](https://github.com/christinac/ellie-slack) - Eliza's younger, hipper, psychobabble-friendly successor as a Slackbot
```
Comment (matiassingers) [inline README.md]: Can we update to `ellie-slack`?

File: README.md [hunk 5]
```diff
@@ -357,8 +357,8 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
...
+- [err](https://github.com/errbotio/errbot) - A plugin-based chatbot designed to be easily deployable, extensible and maintainable
```
Comment (matiassingers) [inline README.md]: Can we update to `errbot`?

File: README.md [hunk 6]
```diff
@@ -474,7 +474,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [tickbot](https://github.com/barryf/tickbot) - Simple team time tracking reminder
 - [trebekbot](https://github.com/gesteves/trebekbot) - A Jeopardy! bot for Slack
 - [vip-notify](https://github.com/doejo/vip-notify) - Send VIP deployment notifications to Slack
-- [webistrano\_slack](https://github.com/caiosba/webistrano_slack) - Send Webistrano deploys to Slack
+- [webistrano\_slack](https://github.com/caiosba/slack-webistrano) - Send Webistrano deploys to Slack
```
Comment (matiassingers) [inline README.md]: Can we update to `slack-webistrano`?

**Comment by matiassingers** [comment]
Thanks so much for this!

----

Just realized this is a bot, haha.

**Comment by ReadmeCritic** [comment]
Will update :wink:

**Comment by matiassingers** [comment]
Perfect, thanks a lot!

**Comment by ReadmeCritic** [comment]
:smile:

**Pull request merged**